lib/app.go

```diff
@@ -139,7 +135,7 @@ func appState(app *App, pod *pkgPod.Pod) error {
 	}
 
 	app.State = AppStateCreated
-	createdAt := fi.ModTime().UnixNano()
```

Comment:
yeah, this breaks https://github.com/coreos/rkt/pull/3318/files#diff-6e87d9abadd3d0ece359c1a0d15ef2b5R47 which is expecting nano